src/zoom_type.h

```diff
@@ -53,6 +53,12 @@ DECLARE_POSTFIX_INCREMENT(ZoomLevel)
 typedef SimpleTinyEnumT<ZoomLevel, byte> ZoomLevelByte;
 
 extern ZoomLevelByte _gui_zoom;
+extern ZoomLevelByte _font_zoom;
 #define ZOOM_LVL_GUI (_gui_zoom)
+/*
+ * Invert _font_zoom to be consistent with _gui_zoom and ZOOM_LVL_GUI
+ * (higher numbers = smaller font)
+ */
```

Comment:
How is this consistent with _gui_zoom? Why doesn't ZOOM_LVL_GUI need this 2 - x?

Reply:
`ZOOM_LVL_GUI` has the convention of its value being inversely correlated with the GUI size (a higher value indicates a smaller GUI). This seems to work well for how it's used, which I haven't looked _too_ much into but seems to me like a fixed array of preset sizes.
I had `_font_zoom` follow the same convention for consistency when setting the value but due to the way this value is ultimately used directly as a multiplier it has to be inverted somewhere. 2 is the maximum zoom size.

Reply:
I think you either want to do it the same way as gui zoom is done, or don't use the ZoomLevel enum at all.

ZOOM_LVL_OUT_4X is for making everything smaller, but in your code you use it to mean 4x larger.

Reply:
Makes sense. I inverted it so it matches GUI zoom. I put the changes in a new commit so you could see them more easily, but I can squash the two commits together if you'd prefer.